parser-typechecker/src/Unison/Runtime/ANF.hs

```diff
@@ -477,9 +477,13 @@ data ANormalF v e
 -- Types representing components that will go into the runtime tag of
 -- a data type value. RTags correspond to references, while CTags
 -- correspond to constructors.
-newtype RTag = RTag Word64 deriving (Eq, Ord, Show, Read, EC.EnumKey)
+newtype RTag = RTag Word64
+  deriving stock (Eq, Ord, Show, Read)
```

Comment:
Yeah, `Read` was stock-derived before and after this PR, so that'd be "RTag 42". Seems right to me, especially because of the (informal?) rule that `Read` should agree with `Show`.

Reply:
I double-checked and the Read instance is unused (everything still compiles after removing it), I'm guessing someone just went a little derive-happy so I'd personally advocate for removing it haha.

I know the "expected" behaviour is for it to agree with Show, but in my experience using `Read` for anything other than primitive types is an anti-pattern, better to write a proper parser, or a `String -> Either String a` function IMO

Reply:
Cool. Yeah, I don't use `Read` myself, but I didn't author this code originally. Let's maybe suggest removing it in another PR?